import_module/create_groups.py

```diff
@@ -449,11 +449,15 @@ def create_tasks(xmin, xmax, ymin, ymax, config):
 def create_groups(groups, config):
     # this function will create a json file to upload in firebase groups table
 
+    print(config)
     # Create the output Driver
     outDriver = ogr.GetDriverByName('GeoJSON')
+    print(outDriver)
     # Create the output GeoJSON
-    outfile = 'data/groups_{}.geojson'.format(config["project_id"])
+    outfile = '/data/groups_{}.geojson'.format(config["project_id"])
```

Comment:
@Hagellach37 this is actually the only line that mattered. It was a folder access issue. I would suggest checking paths and folder creations again or transitioning fully to docker maybe?

Reply:
The outDataSource ended up being `None`